test-utils-base/src/commonTest/kotlin/FlowExtensionsTest.kt

```diff
@@ -0,0 +1,166 @@
+/*
+ Copyright 2023 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.test.base
+import com.splendo.kaluga.base.runBlocking
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.coroutines.flow.flowOf
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+import kotlin.time.Duration.Companion.milliseconds
+
+class FlowExtensionsTest {
+    private companion object {
+        val TIMEOUT = 50.milliseconds
```

Comment:
Optimistic for the emulator tests probably. But other tests will run faster and I think it will still pass (just not fail if something is wrong)? correct?

Reply:
yes, you are correct. the test won't capture the part for the State/SharedFlow, however there are "simple" flow checks happening before that